src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -188,7 +195,17 @@ class ContentOpenWith extends PureComponent<Props, State> {
      * @return {boolean}
      */
     isBoxEditIntegration(integrationId: ?string): boolean {
-        return integrationId === BOX_EDIT_INTEGRATION_ID;
+        return integrationId === BOX_EDIT_INTEGRATION_ID || this.isBoxEditSFCIntegration(integrationId);
+    }
+
+    /**
+     * Checks if a given integration is a Box Edit integration.
+     *
+     * @param {string} [integrationId] - The integration ID
+     * @return {boolean}
+     */
+    isBoxEditSFCIntegration(integrationId: ?string): boolean {
+        return integrationId === BOX_EDIT_SFC_INTEGRATION_ID;
```

Comment:
It looks like this logic is now duplicated in the `OpenWith` component. In the future, it seems like it'd be better to have each item in the dropdown decide whether it should be shown rather than having the parent do it.

Reply:
We would still need this logic at the top level to branch on handling successful integrations, as well as deciding what scope to pass to Box Tools. 

If each item decided whether to render or not, it would need knowledge of the other integrations. e.g. as a Box Edit SFC integration I should hide myself if the regular Box Edit integration exists.